binding/form_mapping.go

```diff
@@ -23,6 +24,15 @@ func mapForm(ptr interface{}, form map[string][]string) error {
 
 		structFieldKind := structField.Kind()
 		inputFieldName := typeField.Tag.Get("form")
+		inputFieldNameList := strings.Split(inputFieldName, ",")
+		inputFieldName = inputFieldNameList[0]
+		var defaultValue interface{}
+		if len(inputFieldNameList) > 1 {
+			defaultList := strings.Split(inputFieldNameList[1], "=")
```

Comment:
maybe use `strings.SplitN(inputFieldNameList[1], "=", 2)` and remove the condition below `len(defaultList) == 2 `. as it is right now, is also ok.

https://golang.org/pkg/strings/#SplitN

Reply:
Yes, we should use `SplitN` which can reduce some codes(`interface{}` and `len(defaultList)==2`).